src/Compilers/CSharp/Portable/Symbols/SymbolWithAnnotations.cs

```diff
@@ -169,18 +175,33 @@ internal abstract class TypeSymbolWithAnnotations : NamespaceOrTypeSymbolWithAnn
     {
         public static TypeSymbolWithAnnotations Create(TypeSymbol typeSymbol)
         {
+            if (typeSymbol is null)
+            {
+                return null;
+            }
+
             // TODO: Consider if it makes sense to cache and reuse instances, at least for definitions.
```

Comment:
Not from this PR: the TODOs in this file should be changed to PROTOTYPE

Reply:
Done.